docs/getting-started/quickstart.md

```diff
@@ -5,13 +5,14 @@ sidebar_position: 1
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# DBOS SDK Quickstart
+# DBOS Quickstart
 
 Here's how to get a simple DBOS "Hello, Database!" application up and running in less than five minutes.
+First we'll show you how to run it locally, then we'll show you how to deploy it to DBOS Cloud!
 
-### System Requirements
+## System Requirements
```

Comment:
Maybe the title is just `Installation`, and we say we support macOS, Windows (including WSL), and Linux.
https://nextjs.org/docs/getting-started/installation